rest/client/rest.go

```diff
@@ -120,14 +120,18 @@ func (c *communicatorImpl) ModifySpawnHost(ctx context.Context, hostID string, c
 	return nil
 }
 
-func (c *communicatorImpl) StopSpawnHost(ctx context.Context, hostID string, wait bool) error {
+func (c *communicatorImpl) StopSpawnHost(ctx context.Context, hostID string, subscriptionType string, wait bool) error {
```

Comment:
What will happen if someone has not yet updated their CLI? Conversely, what will happen if we revert the deploy that contains this, and someone has an updated CLI but the app server is unaware of the new fields?

Reply:
It should work out in the second case, they will have an option on their CLI that the server will ignore.

Reply:
fixed.